app/Http/Controllers/Api/V1/C2E/MediaCatalog/MediaCatalogAPISettingsController.php

```diff
@@ -191,6 +192,30 @@ public function destroy(Organization $suborganization, MediaCatalogAPISetting $s
         return response(['message' => $this->mediaCatalogAPISettingRepository->destroy($setting)], 200);
     }
 
+    /**
+     * Get All Media Catalog SRT Content for listing.
+     *
+     * Returns the paginated response with pagination links (DataTables are fully supported - All Params).
+     *
+     * @urlParam apisetting required The Id of a media_catalog_api_settings Example: 1
+     * @queryParam start Offset for getting the paginated response, Default 0. Example: 0
+     * @queryParam length Limit for getting the paginated records, Default 25. Example: 25
+     *
+     * @responseFile 200 responses/c2e/media-catalog/srt-content/srt-contents.json
+     *
+     * @param Request $request
+     * @param MediaCatalogAPISetting $apisetting
+     * 
```

Comment:
Remove line return between 
"@param MediaCatalogAPISetting $apisetting"
and 
"@return MediaCatalogAPISettingCollection"